src/env.h

```diff
@@ -449,6 +482,17 @@ class Environment {
                                 const char* name,
                                 v8::FunctionCallback callback);
 
+  typedef
+  ListHead<WorkerContext,
+           &WorkerContext::subworker_list_member_> WorkerContextList;
+
+  inline uv_mutex_t* ApiMutex();
+  inline bool CanCallIntoJs() const;
+  inline void AddSubWorkerContext(WorkerContext* context);
+  inline void RemoveSubWorkerContext(WorkerContext* context);
+  WorkerContextList* sub_worker_contexts() {return &sub_worker_contexts_;}
```

Comment:
Style: space after {, before }.

For consistency, it'd be better to move it into env-inl.h.